### ISSUE TITLE
Question about Displaying Dice and HD Metrics in 3DUnetCNN during Training/Testing

### ISSUE THREAD

**Issue by harrymagic098** [ellisdg/3DUnetCNN#346]
Hello, I am using your 3DUnetCNN project for medical image segmentation training and testing. I would like to ask whether the project supports calculating and displaying some common metrics during training or testing, such as the Dice coefficient and Hausdorff Distance (HD)? If it's not currently supported, do you have any suggestions on how to implement these? Thank you for your help!

**Comment by ellisdg** [comment]
> However, I’ve encountered a new issue. The label values I retrieve from both the train_loader and val_loader contain decimal numbers between 0 and 1.

It sounds like you have prediction 3 channels after passing the sigmoid activation. If that is true, the next step would be to threshold the values such that (usually at 0.5) to get a binary representation. If this isn't the case, then I might need to see more code to understand the context. If you want to create the predicted labelmap, you will need to do something similar to this [function](https://github.com/ellisdg/3DUnetCNN/blob/master/unet3d/utils/one_hot.py#L101).

> Additionally, I’d like to ask about the model output. Do the three output channels represent the probabilities for the ET, TC, and WT regions, respectively? If so, what would be a suitable threshold for these probabilities?

Assuming you are using the brats configuration in the example, the 3 channels correspond to [hierarchical](https://github.com/ellisdg/3DUnetCNN/blob/master/examples/brats2020/brats2020_config.json#L140) combination of [labels 2, 1, and 4](https://github.com/ellisdg/3DUnetCNN/blob/master/examples/brats2020/brats2020_config.json#L135). This means channel 0 is [2, 1, 4], channel 1 is [1, 4], and channel 2 is [4]. 

Looking at the [BraTS 2020 website](https://www.med.upenn.edu/cbica/brats2020/tasks.html#Task1BrainTumorSegmentationinMRIscans1), [2, 1, 4] is WT, [1, 4] is TC, and [4] is ET.
